scripts/gatsby-source-iiif/src/fetchManifests/fixIiifJson/index.js

```diff
@@ -47,8 +47,11 @@ const fixLanguage = (data) => {
 }
 
 const removeProvider = (data) => {
+  // delete data['@context']
+  // delete data.seeAlso
+
   if (data.provider && (!data.type || data.type === 'Canvas')) {
```

Comment:
Can this whole if statement be removed?

Reply:
not yet.  But if i can't make it work right.  I will.

Reply:
👍